packages/server/graphql/public/types/RemoveMultipleOrgUsersSuccess.ts

```diff
@@ -0,0 +1,41 @@
+import {KickedOutNotification} from '../../../postgres/types/Notification'
+import isValid from '../../isValid'
+import {RemoveMultipleOrgUsersSuccessResolvers} from '../resolverTypes'
+
+export type RemoveMultipleOrgUsersSuccessSource = {
+  orgId: string
+  teamIds: string[]
+  teamMemberIds: string[]
+  taskIds: string[]
+  userIds: string[]
+  kickOutNotificationIds: string[]
+  organizationUserIds: string[]
+}
+
+const RemoveMultipleOrgUsersSuccess: RemoveMultipleOrgUsersSuccessResolvers = {
+  organization: async ({orgId}, _args, {dataLoader}) => {
+    return await dataLoader.get('organizations').loadNonNull(orgId)
+  },
+  teams: async ({teamIds}, _args, {dataLoader}) => {
+    return (await dataLoader.get('teams').loadMany(teamIds)).filter(isValid)
+  },
+  teamMembers: async ({teamMemberIds}, _args, {dataLoader}) => {
+    return (await dataLoader.get('teamMembers').loadMany(teamMemberIds)).filter(isValid)
+  },
+  updatedTasks: async ({taskIds}, _args, {dataLoader}) => {
+    return (await dataLoader.get('tasks').loadMany(taskIds)).filter(isValid)
+  },
+  users: async ({userIds}, _args, {dataLoader}) => {
+    return (await dataLoader.get('users').loadMany(userIds)).filter(isValid)
+  },
+  kickOutNotifications: async ({kickOutNotificationIds}, _args, {dataLoader}) => {
+    return (await dataLoader.get('notifications').loadMany(kickOutNotificationIds)).filter(
```

Comment:
-1 these are filtered by `userId == viewerId` in `RemoveOrgUserPayload`. I'm not sure what this is for, because the viewer should already have received a notification if they're affected.